### ISSUE TITLE
"UNVR:ssh.run[uname]" became not supported: Cannot access public key file /var/lib/zabbix/.ssh/zb_id_rsa/zb_id_rsa.pub

### ISSUE THREAD

**Issue by justchil** [patricegautier/unifiZabbix#70]
I will try to describe this issue the best I can.

{$UNIFI_SSH_PRIV_KEY_PATH}  /var/lib/zabbix/.ssh/zb_id_rsa

The path being the path/file is required to make unifi templates work.  It uses -i {$UNIFI_SSH_PRIV_KEY_PATH} on the mca-dump-short.

However the UNVR requires the high priority SSH host template.   It uses both the key path {$UNIFI_SSH_PRIV_KEY_PATH} and {$UNIFI_PUB_KEY}.  

290732:20221112:173523.857 item "UNVR:ssh.run[uname]" became not supported: Cannot access public key file /var/lib/zabbix/.ssh/**zb_id_rsa**/zb_id_rsa.pub

I thought I could add macros and set values on the UNVR to override those values.  It's not working though.  Any ideas on how I can fix this?  I'm still fairly new to Zabbix but I'm trying my best.

**Comment by patricegautier** [comment]
Hi -
What values do you have for  {$UNIFI_SSH_PRIV_KEY_PATH}  and {$UNIFI_PUB_KEY}?

I have:

zb_id_rsa.pub for  {$UNIFI_PUB_KEY}
/var/lib/zabbix/ssh_keys/zb_id_rsa for {$UNIFI_SSH_PRIV_KEY_PATH}

**Comment by patricegautier** [comment]
I am assuming those are the global values and they look right.. Is there a local override then on the nvr host itself in zabbix?

for what it's worth I have only one override on NVRs and it is

{$UNIFI_USER} set to root

**Comment by RainCaster** [comment]
I have a UNVR4 and I can connect easily. I did have some permission issues at first. 

1. First, I needed to have the keys completely accessible under the "zabbix" user context. 
2. Then I needed to create the directory /var/lib/zabbix/.ssh 
3. Create a file within there named "known_hosts" 
4. sudo chmod 666 /var/lib/zabbix/.ssh/known_hosts
Yeah- that last is isn't great from a a security POV.

My unifi setup has different accounts for the CK/UNVR and the Switch/AP devices. I'm thinking that the package should be updated to provide macros for each independently in case others have their stuff set up like I do.
